### PR TITLE
32/add mentions translations

### LINKED ISSUE #32: add mentions translations

### PR DESCRIPTION
**Pull request by spaenleh**
This PR adds translation keys for the chat mentions.

closes #32

### CODE REVIEW

File: src/langs/en/chatbox.json [hunk 1]
```diff
@@ -4,5 +4,18 @@
...
+  "CHATBOX_HEADER": "Chatbox",
+  "INPUT_MESSAGE_TOO_LONG": " (max. {{length}} chars)",
+  "SUGGESTED_MENTIONS": "Suggested mentions",
+  "ANONYMOUS_USER": "Anonymous",
+  "DELETE_TOOLTIP": "Delete",
+  "MARK_AS_READ": "Mark as read",
+  "MARK_ALL_AS_READ": "Mark all as read",
+  "CLEAR_ALL": "Clear all",
+  "CLEAR_ALL_NOTIFICATIONS_TITLE": "Clear All Notifications",
+  "CLEAR_ALL_NOTIFICATIONS_CONTENT": "Are you sure you want to clear all your notifications ?",
```
Comment (pyphilia) [inline src/langs/en/chatbox.json:16]:
```suggestion
"CLEAR_ALL_NOTIFICATIONS_CONTENT": "Are you sure you want to clear all your notifications?",
```

File: src/langs/fr/chatbox.json [hunk 2]
```diff
@@ -4,5 +4,18 @@
...
   "INPUT_FIELD_PLACEHOLDER": "Écrivez quelque chose…",
-  "CHATBOX_HEADER": "Chatbox"
+  "CHATBOX_HEADER": "Chatbox",
+  "INPUT_MESSAGE_TOO_LONG": " (max. {{length}} caractères)",
+  "SUGGESTED_MENTIONS": "Mentions suggérées",
+  "ANONYMOUS_USER": "Anonyme",
+  "DELETE_TOOLTIP": "Supprimer",
+  "MARK_AS_READ": "Marquer comme lu",
+  "MARK_ALL_AS_READ": "Marquer tous comme lu",
+  "CLEAR_ALL": "Supprimer toutes",
```
Comment (pyphilia) [inline src/langs/fr/chatbox.json:14]: `Tout supprimer`?

File: src/langs/fr/chatbox.json [hunk 3]
```diff
@@ -4,5 +4,18 @@
...
+  "CLEAR_ALL_NOTIFICATIONS_TITLE": "Supprimer toutes les notifications",
+  "CLEAR_ALL_NOTIFICATIONS_CONTENT": "Êtes vous sûr de vouloir supprimer toutes vos notifications ?",
```
Comment (pyphilia) [inline src/langs/fr/chatbox.json:16]:
```suggestion
"CLEAR_ALL_NOTIFICATIONS_CONTENT": "Êtes-vous sûr de vouloir supprimer toutes vos notifications ?",
```

File: src/langs/fr/chatbox.json [hunk 4]
```diff
@@ -4,5 +4,18 @@
...
+  "COL_STATUS": "Statut",
+  "COL_MESSAGE": "Message",
+  "COL_BY": "By",
```
[the remark this answers is not in the document]
Comment (spaenleh) [reply src/langs/fr/chatbox.json:19]: 😅 woops thank you for spotting that !

**Review by pyphilia — APPROVED** [review]

**Pull request merged**